code/modules/mob/living/living_defines.dm

```diff
@@ -120,3 +120,15 @@
 	var/slowed_by_drag = TRUE //Whether the mob is slowed down when dragging another prone mob
 
 	var/is_busy = FALSE //Used for random actions that take time. ex: curbstomping. We need to make sure we can only do one of these at a time.
+
+	var/obj/chattext/chattext = new
+	
+	var/last_heard_name = null
+	var/last_used_color = null
+
+obj/chattext
+	var/list/chats = list()

```

Comment:
We really don't need list bloat.

Reply:
Not that this can be helped, I'm just lamenting out tons and tons and tons of lists eating up all of our memory.

Reply:
we need lists for messages to interact with eachother

Reply:
See previous message.